src/compiler/scala/tools/nsc/typechecker/TypeDiagnostics.scala

```diff
@@ -710,13 +710,11 @@ trait TypeDiagnostics {
           opc.iterator.exists(pair => pair.low == m)
         }
         import PartialFunction._
-        def isConvention(p: Symbol): Boolean = {
-          val ds = currentRun.runDefinitions
-          import ds._ ; (
```

Comment:
Yeah right, like the syntax people are asking for. With the syntax, it could warn when the code was changed to make it unnecessary, as opposed to just unused import.